pkg/pod/script.go

```diff
@@ -205,3 +230,57 @@ cat > ${scriptfile} << '%s'
 func encodeScript(script string) string {
 	return base64.StdEncoding.EncodeToString([]byte(script))
 }
+
+func checkWindowsRequirement(steps []v1beta1.Step, sidecars []v1beta1.Sidecar) bool {
+	requiresWindows := false
+	// Detect windows shebangs
+	for _, step := range steps {
+		cleaned := strings.TrimSpace(step.Script)
+		if strings.HasPrefix(cleaned, "#!win") {
+			requiresWindows = true
```

Comment:
Could this just be `return true`?

Same below with the `requiresWindows = true; break` which just does `return requiresWIndows`.

Reply:
Yes, this should just be return true. The variable is an artifact from before I refactored this into a helper method, I'll change that :)